src/Microsoft.AspNetCore.SignalR.Core/DefaultHubLifetimeManager.cs

```diff
@@ -172,20 +176,25 @@ public override Task SendGroupAsync(string groupName, string methodName, object[
         public override Task SendGroupsAsync(IReadOnlyList<string> groupNames, string methodName, object[] args)
         {
             // Each task represents the list of tasks for each of the writes within a group
-            var tasks = new List<Task>();
-            var message = CreateInvocationMessage(methodName, args);
+            List<Task> tasks = null;
+            var message = CreateSerializedInvocationMessage(methodName, args);
 
             foreach (var groupName in groupNames)
             {
                 if (string.IsNullOrEmpty(groupName))
                 {
-                    throw new ArgumentException(nameof(groupName));
+                    throw new InvalidOperationException("Cannot send to an empty group name.");
```

Comment:
Updated exception because the current one won't make sense to the caller (there is no argument called `groupName`). Suggestions for a better message?